packages/notte-browser/src/notte_browser/controller.py

```
@@ -92,11 +93,23 @@ async def execute_browser_action(self, window: BrowserWindow, action: BaseAction
         return True
 
     async def execute_interaction_action(self, window: BrowserWindow, action: InteractionAction) -> bool:
```

Pylint (3.3.7)

[refactor] Too many branches (20/12)

(R0912)